<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1480</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2" rowspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>450</height>
       </size>
      </property>
      <property name="title">
       <string>相机操作</string>
      </property>
      <widget class="QPushButton" name="pushButton_saveImg">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>70</width>
         <height>50</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>保存图像</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pauseButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>80</y>
         <width>70</width>
         <height>50</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string>播放控制</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_sysTime">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>系统时间</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" rowspan="3">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>500</height>
       </size>
      </property>
      <property name="title">
       <string>控制智能车</string>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>29</y>
         <width>361</width>
         <height>181</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="2">
         <widget class="QPushButton" name="pushButton_turnRight">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>右转</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton_clearCommand">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>清除命令</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButton_goFront">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>前进</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_turnLeft">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>左转</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_CARconnect">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>60</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>70</height>
           </size>
          </property>
          <property name="text">
           <string>连接</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="pushButton_stop">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>65</height>
           </size>
          </property>
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="pushButtonCARBACK">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>65</height>
           </size>
          </property>
          <property name="text">
           <string>后退</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="pushButton_disconnect">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>65</height>
           </size>
          </property>
          <property name="text">
           <string>断开连接</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" rowspan="8" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,20,1">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
        <item>
         <widget class="QComboBox" name="urlList">
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>rtsp://192.168.3.66:8554/test</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rtsp://192.168.3.88:8554/test</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rtsp://192.168.3.20:8554/test</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>rtsp://admin:Admin123@192.168.1.246:554/h264/ch1/sub/av_stream</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startButton">
          <property name="text">
           <string>Open</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QDisplay" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>640</width>
          <height>480</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>2160</width>
          <height>1215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="8" column="2">
     <widget class="QTextBrowser" name="textBrowser_log">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QComboBox" name="comboBox_ipAddr"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1480</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QDisplay</class>
   <extends>QWidget</extends>
   <header>qdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
